testData/solutions/win32.json

```diff
@@ -1426,8 +1786,9 @@
                     "highContrastEnabled": "http://registry\\.gpii\\.net/common/highContrastEnabled",
                     "highContrastTheme": "http://registry\\.gpii\\.net/common/highContrastTheme",
                     "invertColours": "http://registry\\.gpii\\.net/common/invertColours"
-                }
-           }
+                },
+                "inverseCapabilitiesTransformations": {}
```

Comment:
What was the convention we decided on? That writing an empty block will defeat the framework's attempt to write an inversion? Where will we document this convention once we remember what it is - where is our "go-to" place for solutions registry docs? I imagine we had probably decided to put all such stuff in inline READMEs in the repository or so?